src/common/low_precision_transformations/include/low_precision/rt_info/avg_pool_precision_preserved_attribute.hpp

```diff
@@ -24,7 +24,7 @@ namespace ngraph {
  */
 class LP_TRANSFORMATIONS_API AvgPoolPrecisionPreservedAttribute : public PrecisionPreservedAttribute {
 public:
-    OPENVINO_RTTI("LowPrecision::AvgPoolPrecisionPreserved", "", ov::RuntimeAttribute, 0);
+    OPENVINO_RTTI("LowPrecision::AvgPoolPrecisionPreserved", "", ov::RuntimeAttribute);
```

Comment:
`version` is deprecated, and should be replaced by `version_id`, does it make sense?

Reply:
> `version` is deprecated, and should be replaced by `version_id`, does it make sense?

The second parameter is `version_id`.